help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -87,7 +88,7 @@ To configure details for the destination, fill in the required and optional fiel
 ![Platform UI screenshot showing the destination details.](../../assets/catalog/mobile-engagement/line/destination-details.png)
 
 * **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
-* **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+* **[!UICONTROL Description]**: A description that will help you identity this destination in the future.
```

Comment:
@mohan-h  - for replacing identify with identity, I meant this location (line 118)

`If your target identify is *UserID* you will need the below:`